test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/NotExpressionTests.cs

```diff
@@ -0,0 +1,112 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Serialization;
+using Microsoft.Health.Fhir.Client;
+using Microsoft.Health.Fhir.Tests.Common;
+using Microsoft.Health.Fhir.Tests.Common.FixtureParameters;
+using Microsoft.Health.Fhir.Tests.E2E.Common;
+using Microsoft.Health.Test.Utilities;
+using Xunit;
+using ResourceType = Hl7.Fhir.Model.ResourceType;
+using Task = System.Threading.Tasks.Task;
+
+namespace Microsoft.Health.Fhir.Tests.E2E.Rest.Search
+{
+    [Trait(Traits.Category, Categories.Search)]
+    [HttpIntegrationFixtureArgumentSets(DataStore.SqlServer, Format.Json)]
+    public sealed class NotExpressionTests : IClassFixture<HttpIntegrationTestFixture>
```

Comment:
Should missing be added to the class/test names? It looks like that expression is also being tested in this file.

Reply:
That's a good question. 

Originally the bug that I'm fixing is ":not operator not working as expected when combined with other operators.".
The addition of the "missing" operator is just to have the additional operator to force the problem.

The target of the test is not testing the :missing operator, but the usage of :not operator.